src/js/components/Form/Form.js

```diff
@@ -85,15 +85,21 @@ const Form = forwardRef(
       setValue(prevValue => {
         const nextValue = { ...prevValue };
         nextValue[name] = data;
-
         // re-run any validations, in case the validation
         // is checking across fields
         setErrors(prevErrors => {
           const nextErrors = { ...prevErrors };
           Object.keys(prevErrors).forEach(errName => {
             if (validations.current[errName]) {
-              const nextError = validations.current[errName](data, nextValue);
-              updateErrors(nextErrors, errName, nextError);
+              const nextError = validations.current[errName](nextValue, data);
+              if (!nextError && nextErrors[name]) {
```

Comment:
Isn't `nextErrors[name]` always going to be true, since we built `nextErrors` from `prevErrors` at line 91 and are only going through the keys in `prevErrors` at line 92?

Reply:
ill double check

Reply:
No not always. It can come out as undefined.